wordpress-mobile/WordPress-iOS
# Update metadata scripts + existing metadata

Downloads all of the latest metadata from App Store Connect and updates WPiOS scripts to update App Store Connect for two new locales: `es-MX` and `ar-SA` in a similar manner as https://github.com/woocommerce/woocommerce-ios/pull/3352.

**To Test:**
- Run `cd Scripts/fastlane && ./download-metadata.swift`. Note that the release notes have been deleted (because they're not present for `16.5` yet).
- Change the `glotPressWhatsNewKey` to `v16.4-whats-new`. Note that far fewer release notes are missing (just the ones that weren't part of `16.4`.
